src/cli/bin.ts

```diff
@@ -175,6 +286,21 @@ This will output a file named \`report.json\` containing the error messages.`);
       process.exit(0);
     },
   )
+  .check(({ fromVersion, toVersion }) => {
+    const fromVersionIndex = migrationFunctions.findIndex(
+      ([version]) => version === fromVersion,
+    );
+    const toVersionIndex = migrationFunctions.findIndex(
+      ([version]) => version === toVersion,
+    );
+    if (fromVersionIndex >= toVersionIndex) {
+      throw new Error(`
+        You specified ${fromVersion} as the version to migrate your content from, and ${toVersion} as the version to migrate your content to.
+        The version to migrate to must be higher than the version to migrate from.
+      `);
```

Comment:
I would keep it simple:
```suggestion
      throw new Error("--to-version must be greater than --from-version");
```